app/models/order.rb

```diff
@@ -1,6 +1,7 @@
 class Order <ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
 
+	belongs_to :user
   has_many :item_orders
```

Comment:
this is one of the relationships that I need to get my part done.

Reply:
+1